src/core/Services/products.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * @author ampats11 (Jeremy Andy F. Ampatin)
+ */
+
+import type {Response, Request} from "express"
+import { result } from "../../utils/response";
+import Product from "../../db/models/product";
+
+
+export function product(request: Request, response: Response){
```

Comment:
I'll leave this pull request open until you add implementation for getting product data.